templates/kube-controller-manager.yaml

```diff
@@ -20,6 +20,8 @@ spec:
     - --service-account-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
     - --root-ca-file=/etc/kubernetes/ssl/ca.pem
     - --leader-elect=true
+    - --cluster-cidr={{kubernetes_cluster_cidr}}
```

Comment:
Can we have a boolean variable `kubernetes_enable_cni` so these options are added only when it is set to `true`?